src/scripts/pokemons/PokemonList.ts

```diff
@@ -471,6 +526,140 @@ const pokemonList = createPokemonArray(
         'exp': 112,
         'eggCycles': 10,
         'evolutions': [new StoneEvolution('Pikachu', 'Raichu', GameConstants.StoneType.Thunder_stone)],
+        //TODO add Alola-only evolution into Alolan Raichu using Thunderstone
+        'base': {
+            'hitpoints': 35,
+            'attack': 55,
+            'specialAttack': 50,
+            'defense': 40,
+            'specialDefense': 50,
+            'speed': 90,
+        },
+    },
+    {
+        'id': 25.01,
+        'name': 'Pikachu (Original cap)',
+        'catchRate': 190,
+        'type': [PokemonType.Electric],
+        'levelType': LevelType.mediumfast,
+        'exp': 112,
+        'eggCycles': 10,
+        'evolutions': [new StoneEvolution('Pikachu', 'Raichu', GameConstants.StoneType.Thunder_stone)],
+        //TODO add Alola-only evolution into Alolan Raichu using Thunderstone
```

Comment:
I think we shouldn't have evolutions for the event Pokémon without also making the evolution follow the same sort of sprite (such as the caps in this case), could we just remove all the evolutions from these cap Pikachus?
```suggestion
```

Reply:
I have no time to fix it now but yeah, all the cap pikachus are not supposed to evolve at all.